app/player/[uuid]/flips/page.tsx

```diff
@@ -105,15 +105,22 @@ export function getEmbedDescription(flipTrackingResponse: FlipTrackingResponse,
             profitByFinderEmbed += `${finderGroup.label}: ${numberWithThousandsSeparators(finderGroup.profit)} Coins \n`
         }
     })
+
     let highestProfitFlip = flipTrackingResponse.flips.sort((a, b) => b.profit - a.profit)[0]
     let highestProfitFlipText = `Highest Profit Flip: ${numberWithThousandsSeparators(highestProfitFlip.profit)} Coins \n ${numberWithThousandsSeparators(
         highestProfitFlip.pricePaid
     )} Coins ➞ ${numberWithThousandsSeparators(highestProfitFlip.soldFor)} Coins (${removeMinecraftColorCoding(highestProfitFlip.item.name)})`
 
+    let lowestProfitFlip = flipTrackingResponse.flips.sort((a, b) => b.profit - a.profit)[flipTrackingResponse.flips.length-1]
```

Comment:
The array was already sorted like that, just a few lines before.
Store the sorted array in a variable and just access that, instead of sorting it twice

Reply:
I believe this has been fixed